client/gutenberg/extensions/slideshow/style.scss

```diff
@@ -57,10 +57,8 @@
 		background-repeat: no-repeat;
 		background-size: 24px;
 		border: 0;
-		width: 24px;
-		height: 24px;
-		box-sizing: content-box;
-		padding: 12px;
```

Comment:
Shouldn't we explicitly set `padding: 0;`? Or is this unnecessary?

Reply:
Good catch. Just in case I will add one 😅

Reply:
Fixed in 02b5559